simulaqron/SimulaQron.py

```diff
@@ -121,22 +164,39 @@ def stop(name):
 
 
 @cli.command()
-def reset():
+@click.option(
+    "-f",
+    "--force",
+    help="Don't ask for confirmation.",
+    is_flag=True,
+)
+def reset(force):
     """Resets simulaqron"""
-    for entry in os.listdir(PID_FOLDER):
-        if entry.endswith(".pid"):
-            pidfile = os.path.join(PID_FOLDER, entry)
-            d = SimulaQronDaemon(pidfile=pidfile)
-            d.stop()
-            if os.path.exists(pidfile):
-                os.remove(pidfile)
+    if not force:
+        answer = input("Are you sure you want to reset simulaqron?\nThis will revert settings and "
+                       "network config files to the default values.\nNote, this will not edit or remove "
+                       "the file at ~/.simulaqron.json if it exists, this you have to do manually if "
+                       "you wish to revert all settings.\n"
+                       "(yes/no)")
+    else:
+        answer = "yes"
+    if answer == "yes":
```

Comment:
or answer == "y"

Quicker! And more inline with what's accepted in linux shells,

Reply:
Yes, we can add that, (or answer.startswith('y')?